bin/i18n/resources/pegasus/markdown/sync_in.rb

```diff
@@ -15,7 +15,6 @@ class SyncIn < I18n::Utils::SyncInBase
             public/csforgood.md
             public/curriculum/unplugged.md.partial
             public/educate/it.md
-            public/helloworld.md.partial
```

Comment:
@mgc1194 @castro-jorge is this one of those times where there's a step to clean this up in Crowdin as well?